### PR TITLE
fix: unexpected extra width - UI components for mobile devices

### LINKED ISSUE #995: [UI] Improve The UI for Mobile devices Unexpected Extra width
#### Current Behavior
I've created an issue regarding an issue with uneven spacing and component size on mobile devices when using Meshery. Specifically, it appears that there is extra width present on the mobile layout, potentially causing difficulty for users trying to interact with the interface.

This issue occur when the screen width is below 430px.

#### Desired Behavior

On mobile devices, the dimensions of the components should be adjusted in accordance with the device's screen size while maintaining the relative spacing between elements.

#### Implementation

use CSS media query and decrease the size of the elements a little maintaing the proportions and relative spacing between different components.

#### Acceptance Tests

N/A

#### Mockups

N/A

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by aminoxix**
**Description**
- fixed unexpected extra width by reducing the `min-width` to `360px` means the scrolling issue was fixed on the `x-axis` for users having screen below `400px`
- standardized the navbar with good practice by moving the mode toggle button to `navigation.html`

image

This PR fixes #995 

**Notes for Reviewers**
"We should migrate this to vanilla HTML, CSS & JS to impact more contributors instead Jekyll, if possible!"

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by GaganpreetKaurKalsi — APPROVED** [review]
Things are working fine. Looks good 👍 💯 
Great work done @aminoxix 🥇 Saw the conversation in issue, pretty long pending I would say.
Finally it would get fixed 🚀

**Comment by aminoxix** [comment]
@GaganpreetKaurKalsi what's your insights on the reviewer's note?

**Comment by GaganpreetKaurKalsi** [comment]
@leecalcote Would you like me to exercise my merging powers here?

**Review by thisiskaransgit — COMMENTED** [review]
Thank you @aminoxix, for this update, since this change will affect the responsiveness globally, there are things to consider
- at < 360px in /the features section, there's an overflow issue for the SMP button,  
Screenshot from 2023-05-23 14-14-38
 - Changes in the left margin affect other components
   - table has shifted to left in /talks-and-presentations
   Screenshot from 2023-05-23 14-33-11

   - filter button in /catalog section has moved toward the left, rather than in center
Screenshot from 2023-05-23 14-31-23

Kindly look into it and update the PR and we are good to go.

**Comment by aminoxix** [comment]
Hey @thisiskaransgit,
1. very catchy, I might say I would be more tho as I fixed one more button on the same page! 👀 

2. but, yeah I think according to you & website, it seems `/catalog` page "Filter button" is working fine for me (quite the same as the [deploy preview](https://deploy-preview-1159--mesheryio-preview.netlify.app/catalog)).

image

3. well, the same for `/talks-and-presentations`, do you want me to fix "Find video and slides here" button to `center`? If so, then I think mate, this PR is already ready to merge, and we can track other broken components via another issue. // @leecalcote 

image

Well, I checked it for all mobile screens, seems all looks good!

**Comment by leecalcote** [comment]
> @leecalcote Would you like me to exercise my merging powers here?

@GaganpreetKaurKalsi I missed this note earlier. Yes, if you're comfortable with this change, then please do merge.

File: _includes/header.html [hunk 1]
```diff
@@ -58,21 +58,15 @@
               alt="Meshery Logo"
             />&nbsp;
             <img
-              src="{{ site.baseurl }}/assets/images/logos/meshery-logo-text-dark.png"
-              data-logo-for-dark="{{ site.baseurl }}/assets/images/logos/meshery-logo-text.png"
-              data-logo-for-light="{{ site.baseurl }}/assets/images/logos/meshery-logo-text-dark.png"
-              alt="Meshery Logo"
-              id="logo-dark-light"
```
Comment (thisiskaransgit) [inline _includes/header.html:65]: revert the addition of whitespaces please

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@aminoxix, thank you for incorporating the changes and looking up for additional things to fix.
-  for the `/catalog` section, remove the left padding for the filter wrapper from the media query at about 700 px.

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@aminoxix, Changes made in the header have resulted in the toggle icon being relatively smaller than desired, it would be better to revert the changes made here, 
**Current:** 
Screenshot from 2023-05-26 20-14-49
**Preview:** 
Screenshot from 2023-05-26 20-18-35

- Remove the unnecessary white spaces and extra lines, included in the commits 
- The performance-wrapper is inconsistent for 750-800 px, but there's already a PR open for this and we don't want you to steal their limelight, so please revert this one too.
-  make sure that you're not pushing any unwanted changes, so this can be the last revision.

**Comment by aminoxix** [comment]
> @aminoxix, Changes made in the header have resulted in the toggle icon being relatively smaller than desired, it would be better to revert the changes made here, **Current:** Screenshot from 2023-05-26 20-14-49 **Preview:** Screenshot from 2023-05-26 20-18-35
> 
>     * Remove the unnecessary white spaces and extra lines, included in the commits
> 
>     * The performance-wrapper is inconsistent for 750-800 px, but there's already a PR open for this and we don't want you to steal their limelight, so please revert this one too.
> 
>     * make sure that you're not pushing any unwanted changes, so this can be the last revision.

I came across an issue in the header file where the toggle mode SVG element was located. It should actually be placed within the navigation section. In order to improve the visual aspect, I made adjustments by reducing its size since it appeared too large on the navigation bar and had unnecessary empty space around it.

Response to your reviews:
- I understand that there are unexpected indentations in the source code. Although I made sure to fix unnecessary white spaces and extra lines for the specific lines I worked on, it would be helpful if you could highlight that all spaces in this pull request should be addressed. I'm more than willing to fix them all for you.
- Regarding the performance-wrapper, I addressed the spacing issue because you raised it in this pull request. That's why I mentioned discussing those concerns separately and not within this PR. (already mentioned above in point 3 [here](https://github.com/meshery/meshery.io/pull/1159#issuecomment-1560054253))
- Could you please provide more clarity on what you mean by "unwanted changes"? It caught my attention, and I'm genuinely curious about the specific aspects you are referring to.

**Comment by vishalvivekm** [comment]
@aminoxix 
Let's discuss this on websites call, add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#), if you would :)

**Comment by aminoxix** [comment]
hey @vishalvivekm, thanks for taking this up, but I'm done with this issue now, if anyone wanna work or just approve, it's open! (:

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by thisiskaransgit** [comment]
@leecalcote @Chadha93, we're good to go with this one !

**Comment by aminoxix** [comment]
@thisiskaransgit: https://github.com/meshery/meshery.io/pull/1159#issuecomment-1560073123

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**